git-hg/git-hg.sh

```diff
@@ -16,41 +16,41 @@
 git clone https://github.com/frej/fast-export.git
```

Comment:
Any reason for not setting the `set -eu` at the first line, this will make the `|| exit` and `|| return` unnecessary if we want to script to fail-fast, i would vote for fail-fast. Unless you want a graceful exit for a good reason, there could be disadvantages to graceful exits in some cases.